### PR TITLE
Add benchmarks for f32 & f64 operations.

### PR DESCRIPTION
**Pull request by twitchyliquid64**
```
[xxx@xxx]:~/projects/wagon/exec> go test -bench=BenchmarkF github.com/go-interpreter/wagon/exec
goos: linux
goarch: amd64
pkg: github.com/go-interpreter/wagon/exec
BenchmarkF64Arithmetic10Interpreted-8   	  100000	     14344 ns/op
BenchmarkF64Arithmetic10Native-8        	  300000	      4343 ns/op
BenchmarkF32Arithmetic10Interpreted-8   	  100000	     13365 ns/op
BenchmarkF32Arithmetic10Native-8        	  300000	      4240 ns/op
PASS
ok  	github.com/go-interpreter/wagon/exec	7.688s
```

Looks like the floating point operations are only ~3x faster instead of ~5x. I think this is attributable to missing operations for conversion from integer to float - hopefully I'll get around to that this weekend.

### CODE REVIEW

File: exec/exec_test.go [hunk 1]
```diff
@@ -465,6 +465,46 @@ func BenchmarkU64Arithmetic10Native(b *testing.B) {
...
+		benchmarkDummy, _ = vm.ExecCode(int64(funcIndex), 10, 10)
+	}
+}
+
+func BenchmarkF32Arithmetic10Native(b *testing.B) {
+	if runtime.GOARCH != "amd64" {
+		b.SkipNow()
+	}
+
+	vm, funcIndex := loadModuleFindFunc(b, "testdata/rust-basic.wasm", "loopedArithmeticF64Benchmark", true)
```
Comment (sbinet) [inline exec/exec_test.go]: `s/F64/F32/` ?

Comment (twitchyliquid64) [reply exec/exec_test.go]: ^^'

**Comment by sbinet** [comment]
this should have "improved" the benchmarks, right? :)

**Comment by twitchyliquid64** [comment]
Dont think so, as processors arent much faster on 32bit floats than 64bit floats?

**Comment by sbinet** [comment]
my naive assumption was that pipelines would be somewhat more full.

**Pull request merged**